gradle/wrapper/gradle-wrapper.properties

```diff
@@ -14,9 +14,8 @@
 # limitations under the License.
 #
 
-#Mon Sep 11 11:17:48 PDT 2017
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.10.2-all.zip
```

Comment:
I prefer [gradle-4.10.2-bin.zip](https://stackoverflow.com/questions/25451308/android-studio-gradle-bin-zip-vs-gradle-all-zip)

Reply:
Why do you prefer gradle-4.10.2-bin.zip? The only thing it does it will save a few MB of download a _single time, machine wide_. On the other hand, you won't get the source code and documentation which can be useful when debugging Gradle scripts. Plus, the default setup for new projects generated with Android Studio is the `-all` variant, making it much more likely to be present on your machine already anyway, since most other projects will be using it.
Since this project is already using `-all` as well, I think it makes sense to just stick to it.